README.md

```diff
@@ -115,6 +124,27 @@ file, they can be found in [`scripts`](./docker/package/scripts) and
 [`defaults`](./docker/package/defaults) folders respectively.
 
 
+### Multiple similar systemd services
+
+It's possible to run multiple same services, e.g. two `tezos-node`s that run different
+networks.
+
+`tezos-node` packages provide three services out of the box:
+`tezos-node-delphinet`, `tezos-node-carthagenet` and `tezos-node-mainnet` that run
+`delphinet`, `carthagenet` and `mainnet` networks respectively.
+
+In order to start it run:
+```
+systemctl start tezos-node-<network>
+```
+
+Another case for running multiple similar systemd services is when one wants to have
+multiple daemons that target different protocols.
+Since daemons for different protocols are provided in the different packages, they will
+have different service files. The only thing that needs to be changed is config file.
+One should provide desired node address, data directory for daemon files and node directory
```

Comment:
Here you mean files like `tezos-accuser.conf`, right? How are people supposed to find these files in their Ubuntu/Fedora systems? If there is some standard location for `systemd` configs and it's some basic knowledge for systemd users, feel free to ignore the last question.

Reply:
It's described in the `### Systemd units on Ubuntu or Fedora` paragraph